uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/app.js

```diff
@@ -11,17 +11,21 @@ define([
   'leave-absences/absence-tab/components/absence-tab-entitlements',
   'leave-absences/absence-tab/components/absence-tab-work-patterns',
   'leave-absences/absence-tab/components/contract-entitlements',
+  'leave-absences/absence-tab/components/annual-entitlements',
   'leave-absences/shared/components/staff-leave-calendar',
   'leave-absences/shared/models/calendar-model',
   'leave-absences/shared/models/leave-request-model',
   'leave-absences/shared/modules/shared-settings',
-  'leave-absences/shared/models/absence-type-model'
+  'leave-absences/shared/models/absence-type-model',
+  'leave-absences/shared/models/entitlement-model',
+  'leave-absences/shared/modules/shared-settings'
 ], function (angular) {
   angular.module('absence-tab', [
     'ngResource',
     'ui.bootstrap',
     'common.angularDate',
     'common.directives',
+    'common.mocks',
```

Comment:
@AkA84 could you please take a look as it seems that my explanation above is not comprehensive enough. CC @deb1990